src/types/guilds/explicit_content_filter_level.ts

```diff
@@ -0,0 +1,6 @@
+/** https://discord.com/developers/docs/resources/guild#guild-object-explicit-content-filter-level */
+export enum ExplicitContentFilterLevel {
```

Comment:
Similarly, prefix the enum with `Discord` similar to how each type (channel types, message types) are prefixed with Discord.